.github/workflows/coverage.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+name: Coverage CI
+
+on:
+  [push, pull_request]
```

Comment:
I think just pull_request here is enough, we currently don't have any automatic deploy in place, so, CI result on a push is irrelevant.